src/main/java/net/geforcemods/securitycraft/blocks/reinforced/BaseReinforcedBlock.java

```diff
@@ -60,6 +90,29 @@ public BaseReinforcedBlock(SoundType soundType, Material mat, Supplier<Block> vB
 		vanillaBlockSupplier = vB;
 	}
 
+	@Override
+	public void onEntityCollision(BlockState state, World world, BlockPos pos, Entity entity) {
+		if (this == SCContent.REINFORCED_COBWEB.get())
+			if (entity instanceof PlayerEntity)
+			{
+				TileEntity te = world.getTileEntity(pos);
+
+				if(te instanceof OwnableTileEntity)
+				{
+					OwnableTileEntity ownableTe = (OwnableTileEntity) te;
```

Comment:
I'm sorry for my inconsistent knowledge in this field, but how should I do that? I know there's some easy way but I don't know it, and you can also just tell me where this already happens, so I'll copy from that.

Reply:
Well, you're only using this variable once somewhere else. Just replace that occurance with what you assign to this variable. I understand I have probably worded my comment incorrectly. I meant that casting _here_ is not necessary. Instead, just write `((OwnableTileEntity)te)` in the if statement below instead of `ownableTe`.

Reply:
Ok I was stupid, I previously tried with `(OwnableTileEntity)te.getOwner()` and it sent me an error because it didn't find `getOwner`, as I missed the brackets around the `(OwnableTileEntity)te`.